src/components/avatar/avatar.ts

```diff
@@ -0,0 +1,61 @@
+import Vue, { PluginObject } from 'vue';
+import Component from 'vue-class-component';
+import { Prop } from 'vue-property-decorator';
+import { AVATAR_NAME } from '../component-names';
+import IconPlugin from '../icon/icon';
+import WithRender from './avatar.html?style=./avatar.scss';
+
+const SMALL_AVATAR_SIZE: number = 32;
+
+@WithRender
+@Component
+export class MAvatar extends Vue {
+
+    @Prop({ default: SMALL_AVATAR_SIZE })
+    pixelSize: number;
```

Comment:
pk pas juste un size ici?

Reply:
pis pourquoi se limiter à une taille par pixel? En HTML on s'Attend souvent à faire size="32px/%/em*/etc"

Reply:
J'ai ajouté pixel juste pour connaître l'unité utilisé. J'avais pensé à mettre uniquement une string mais c'est dans les besoins de pouvoir changer automatiquement l'icône par défaut si sa grosseur est plus petite que 32px.

Reply:
Je comprend que c'est flexible de faire une props en px mais en regardand le storybook et le code de avatar (avec le this.pixelSize <= SMALL_AVATAR_SIZE)

je me demande si ca aurait pas mieux de simplement faire une props avec un enum small et large. Si besoin il y a on pourrait ensuite bonifier par une props size qu'en penses-tu?

Reply:
C'est vrai que ça serait le best :)

Reply:
En fait dans Brio me faut déjà un enum pour petit moyen et large (32, 64 et 192) je pourrais les prendre. C'était ma première idée mais je voulait pas trop limiter :P

Reply:
Peut-être laisser les gens définir la largeur en pixels, mais avoir un enum qui propose des tailles standard. Le meilleur des deux mondes ? 
Au niveau du besoin actuel, j'ai vérifié avec JPM et Marie et les 3 tailles standards suffisent, mais c'est parfois difficile de prévoir l'avenir...

Reply:
Si on laisse les gens choisir la taille on peut se retrouver avec des %, em, etc. ce qui m'empêche de connaître la taille actuel si je veux modifier l'avatar par défaut (this.pixelSize <= SMALL_AVATAR_SIZE).